WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersPeriodViewController.swift

```diff
@@ -160,6 +164,32 @@ private extension StoreStatsAndTopPerformersPeriodViewController {
         }
     }
 
+    /// Observe and react to visibility events for the in-app feedback card.
+    func configureInAppFeedbackCardViews() {
+        guard viewModel.canDisplayInAppFeedbackCard else {
+            return
+        }
+
+        _ = viewModel.isInAppFeedbackCardVisible.subscribe { [weak self] isVisible in
```

Comment:
I thought that the observation token had to be retained in order for the subscription to be alive, but after checking the implementation I see that the closure is saved within the subject. 👍 

My only concern would be, isn't this different on how combine works? I think I remember you have to store the `cancelable` object after `sink` in order to keep the subscription active. I may be wrong tho! Haven't really used Combine 😅